scripts/db_update.php

```diff
@@ -76,33 +87,44 @@ function executeSQL($file)
         $sql = file_get_contents($file);
         $db->exec($sql);
 
-        return TRUE;
+        return true;
 
     }
 
-    return FALSE;
+    return false;
 }
 
 /**
  * Return true if specified table exists in database.
- * @param string $tableName
+ *
+ * @param string $tableName - The name of the table to check presence of.
+ *
  * @return bool
  */
-function checkTableExists($tableName)
+function checkTableExists(string $tableName): bool
```

Comment:
I thought this didn't work in PHP < 8?

Reply:
Typed parameters and return values are definitely supported in PHP 7.
There are some special cases like union types (e.g. string || null $param), which require PHP8.
I have carried out tests of this change in PHP 7.4 and 8.1.

Reply:
Okay. Good to know.

Reply:
Sorry, I had the union types slightly wrong. They are actually `string|null $param`. I haven't used any in this script, but I did try to sneak one in when I added the ConfigurationAdmin page.

Reply:
You are a sneaky sneaker who sneaks!